configurations/ldap-authenticator.yaml

```diff
@@ -0,0 +1,7 @@
+authenticator: 'com.scylladb.auth.SaslauthdAuthenticator'
```

Comment:
Let's add another job that use this configuration.

Reply:
> Let's add another job that use this configuration.

Done. just appended a new pipeline.